app/components/user-status.test.js

```diff
@@ -0,0 +1,38 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | user-status', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('Should show I am doing a task', async function (assert) {
+    await render(hbs`
+      <user-status />
+    `);
+
+    assert
+      .dom('[data-test-status]')
```

Comment:
Not installed this add-on yet? 🤔 
https://github.com/simplabs/ember-test-selectors

Reply:
@whyDontI  I was wondering if we could get away with "data-test-button" only instead of using "data-test-button-text" also?

Reply:
Yes, I'll update the tests. Didn't know the designs were updated.